### DISCUSSION TITLE
How to limit wagtail api v2 to a single model by default ?

### DISCUSSION THREAD

**Discussion by p0lygun** [wagtail/wagtail#9574] [General]
Hi :wave: , I would like to know how to limit the v2 api to a single model when using `PagesAPIViewSet`

i can achieve something close to it by using `BaseAPIViewSet` and doing

```py
class ExperiencePageAPIViewSet(BaseAPIViewSet):
    """Class that defines how page api works."""

    renderer_classes = [JSONRenderer]
    model = ExperiencePage
    filter_backends = PagesAPIViewSet.filter_backends
    body_fields = [
        "url",
        "id",
        "full_url",
        "title",
        "description",
        "cover_img_url",
        "code",
        "exp_url",
        "featured",
        "bugged",
    ]
    listing_default_fields = body_fields
```
but then I lose the ability to use Custom Serializer, 

What I'm trying to do is this by default
![image](https://user-images.githubusercontent.com/22869882/199491094-b382aef7-4dbe-402e-a2d0-e0d7254551a7.png)

**Comment by lb-** [comment]
This should be much easier, and documented, in Wagtail 5.2  I think. 

See merged PR https://github.com/wagtail/wagtail/pull/10902
